src/init.c

```diff
@@ -686,22 +686,26 @@ void _julia_init(JL_IMAGE_SEARCH rel)
 #endif
 #endif
 
-#if defined(JL_USE_INTEL_JITEVENTS)
+#if \
+    defined(JL_USE_INTEL_JITEVENTS) || \
```

Comment:
Can we fold this into one flag `JL_USE_JITEVENTS` and set it in `Make.inc`

Reply:
That's doable, but it would be an extra flag for someone just wanting one external profiler enabled.  That would also need to be updated in the documentation.

I'm willing to make the change, however.

Reply:
I was more thinking of hoisting this logic from the source code to the Makefile, since I am not a fan for complicated pre-processor logic, but it would move the logic further away from where it is needed.

@vtjnash any opinion?

Reply:
This preprocessor gating seems unnecessary

Reply:
This was a relatively quick fix to keep the multiple *jit_profiling assignments from causing errors when multiple profilers are enabled.

Could we create just one JL_USE_JITEVENTS that enables all three rather than specifying which profiler?

Reply:
So to enable external profiling you would have to set both `USE_JITEVENTS=1` plus one of the existing three existing flags in `Make.user` or is the idea that `init.c` would compute the value of `USE_JITEVENTS`?

Reply:
My idea would be to just have `JL_USE_JITEVENTS` enable all three profilers instead of an `#if defined(JL_USE_PROFILE_JITEVENTS)` for each one.